storage/Utils/RegionImpl.cc

```diff
@@ -71,7 +71,10 @@ namespace storage
     unsigned long long
     Region::Impl::get_end() const
     {
-	// TODO check if region is empty
+#if 0
+	if (empty())
+	    ST_THROW(Exception("empty region"));
+#endif
```

Comment:
Why is it not possible to enable this check?

Reply:
Because it breaks tests in yast2-storage-ng.